tutorials/mnist_tutorial_tf.py

```diff
@@ -80,39 +80,36 @@ def evaluate():
     model_train(sess, x, y, predictions, X_train, Y_train,
                 evaluate=evaluate, args=train_params)
 
-    # Craft adversarial examples using Fast Gradient Sign Method (FGSM)
-    adv_x = fgsm(x, predictions, eps=0.3)
-    eval_params = {'batch_size': FLAGS.batch_size}
-    X_test_adv, = batch_eval(sess, [x], [adv_x], [X_test], args=eval_params)
-    assert X_test_adv.shape[0] == 10000, X_test_adv.shape
+    # Initialize the Fast Gradient Sign Method (FGSM) attack object and graph
+    FGSM = FastGradientMethod(model, sess=sess, params={'eps': 0.3})
```

Comment:
I updated the object names for attacks to be lowercase in the 4 tutorials (see 7a78ba1)